dataland-frontend/tsconfig.app.json

```diff
@@ -13,10 +13,8 @@
       "ESNext",
       "DOM"
     ],
-    // These override some properties in the tsconfig.web.json (and will likely break <script setup> clauses
-    // But are required as otherwise the openApi client won't compile properly
-    "importsNotUsedAsValues": "remove",
-    "preserveValueImports": false,
+    // Setting the following to true breaks the imports of the auto-generated clients
+    "verbatimModuleSyntax": false,
```

Comment:
Was sind diese Variablen die man auf false setzen muss? (Hier und auch in der tsconfig.commonjs.json Datei)

Reply:
Um die hier geht es: "verbatimModuleSyntax": false,
Das ist per default auf true und dann geht alles kaputt

Reply:
ok